README.md

```diff
@@ -183,9 +183,13 @@ Specifies whether to add a header message at the top of the destination file. Va
 If you set 'warn' to 'true', `concat` adds the following message:
 
 ~~~
-# This file is managed by Puppet. DO NOT EDIT.
+# This file is managed by Puppet. DO NOT EDIT.\n
```

Comment:
I wanted to note the fact that this string is copied verbatim to the file. I could leave off the `\n` and instead s/following message/following line/ in the paragraph above?


> On 29 May 2015, at 17:18, Igor Galić <notifications@github.com> wrote:
> 
> In README.md <https://github.com/puppetlabs/puppetlabs-concat/pull/320#discussion_r31339621>:
> 
> > @@ -183,9 +183,13 @@ Specifies whether to add a header message at the top of the destination file. Va
> >  If you set 'warn' to 'true', `concat` adds the following message:
> >
> >  ~~~
> > -# This file is managed by Puppet. DO NOT EDIT.
> > +# This file is managed by Puppet. DO NOT EDIT.\n
> not really sure how useful the \n here is in the README…
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/puppetlabs/puppetlabs-concat/pull/320/files#r31339621>.
>